jekyll/_cci2/orb-author.md

```diff
@@ -9,285 +9,290 @@ version:
-          steps:
-            - foo/hello:
-                username: "Anna"
+{:.tab.scripts.Orb-Development-Kit-Packing}
+```yaml
+parameters:
+  to:
+    type: string
+    default: "World"
+    description: "Hello to whom?"
+steps:
+  - run:
+      environment:
+        PARAM_TO: <<parameters.to>>
+      name: Hello Greeting
+      command: <<include(scripts/greet.sh)>>
 ```
-{% endraw %}
 
-Please note that `examples` can contain multiple keys at the same level as `simple_greeting`, allowing for multiple examples.
+##### Why include scripts?
+{:.no_toc}
 
-### Expected Usage Results
+CircleCI configuration is written in `YAML`. Logical code such as `bash` can be encapsulated and executed on CircleCI through `YAML`, but, for developers, it is not convenient to write and test our programatic code within a non-executable format. Also, parameters can become cumbersome in more complex scripts as the `<<parameter>>` syntax is a CircleCI native YAML enhancement, and not something that can be interpreted and executed locally.
 
-The above usage example can be optionally supplemented with a `result` key, demonstrating what the configuration will look like after expanding the orb with its parameters:
+Using the orb development kit and the `<<include(file)>>` syntax, you can import existing scripts into your orb, locally execute and test your orb scripts, and even utilize true testing frameworks for your code.
 
-{% raw %}
-```yaml
-version: 2.1
+##### Using parameters with scripts
+{:.no_toc}
+
+To keep your scripts portable and locally executable, it is best practice to expect a set of environment variables within your scripts and set them at the config level. The `greet.sh` file which was included with the special `<<include(file)>>` syntax above in our `greet.yml` command file looks like this:
 
-examples:
-  simple_greeting:
-    description: Greeting a user named Anna
-    usage:
-      version: 2.1
-      orbs:
-        foo: bar/foo@1.2.3
-      jobs:
-        build:
-          machine: true
-          steps:
-            - foo/hello:
-                username: "Anna"
-    result:
-      version: 2.1
-      jobs:
-        build:
-          machine: true
-          steps:
-          - run:
-              command: echo Hello Anna
-      workflows:
-        version: 2
-        workflow:
-          jobs:
-          - build
+{:.tab.greet-script.greet-sh}
+```bash
+echo Hello "${PARAM_TO}"
 ```
-{% endraw %}
 
-### Usage Examples Syntax
-The top level `examples` key is optional. Usage example maps nested below it can have the following keys:
+This way you can both mock and test your scripts locally.
 
-- **description:** (optional) A string that explains the example's purpose, making it easier for users to understand it.
-- **usage:** (required) A full, valid config map that includes an example of using the orb.
-- **result:** (optional) A full, valid config map demonstrating the result of expanding the orb with supplied parameters.
+#### Orb Packing
```

Comment:
Do we need this section here if its automated? we could just link to the orb packing section of the orb-concepts page - and include any additional info from here in that page?